src/Oceananigans.jl

```diff
@@ -208,6 +207,7 @@ include("TurbulenceClosures/TurbulenceClosures.jl")
 include("Forcings/Forcings.jl")
 
 include("ImmersedBoundaries/ImmersedBoundaries.jl")

```

Comment:
Note that Coriolis introduces immersed boundary concepts so we are a little out of order here

Reply:
I would argue that they are grid concepts (`peripheral_node` exists also for a non immersed grid)